src/Utilities/CssConcatenator/RuleBlock.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Pelago\Emogrifier\Utilities\CssConcatenator;
+
+/**
+ * This class represents a CSS rule block.
+ *
+ * @internal
+ *
+ * @author Jake Hotson <jake.github@qzdesign.co.uk>
+ * @author SignpostMarv
```

Comment:
@SignpostMarv I'd like to put your real name here instead of a nickname (and also an email address if that's okay with you). What name would you like to have here, and which email address (if any)?

Reply:
I'm afraid my "real name" only goes on my day job code, and even the it's not my *legal name* I'm using.

Reply:
@SignpostMarv Do you have a "preferred name"?

Reply:
"SignpostMarv"

Reply:
:-) Okay, then I'll keep it.

Reply:
you can either view it as a camel-cased mononym, or the use of "Signpost" as a prefixing honorific to a mononym with a non-existent zero-width space separating the two. I tend to lean towards the former these days.